packages/expo-screen-capture/android/src/main/java/expo/modules/screencapture/ScreenCaptureModule.kt

```diff
@@ -91,8 +93,10 @@ class ScreenCaptureModule : Module() {
       return
     }
     if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE) {
-      currentActivity.registerScreenCaptureCallback(currentActivity.mainExecutor, screenCaptureCallback!!)
-      isRegistered = true
+      if (safeCurrentActivity != null) {
+        safeCurrentActivity!!.registerScreenCaptureCallback(currentActivity.mainExecutor, screenCaptureCallback!!)
+        isRegistered = true
+      }
```

Comment:
We should be able to avoid the explicit null check with
```suggestion
    safeCurrentActivity?.registerScreenCaptureCallback(currentActivity.mainExecutor, screenCaptureCallback!!)
```

Reply:
Yeah, but then we'd set `isRegistered` to true even though the registration didn't go through?

Reply:
Sorry, I thought of that and updated the comment. See the new version. It's not a big deal. Feel free to merge if you want to leave as is 👍

Reply:
Good point using the Elvis-operator there - changed this.